migrator.go

```diff
@@ -33,6 +33,14 @@ type ViewOption struct {
 	Query       *DB
 }
 
+// ALLTables get all database tables
```

Comment:
maybe just add a new method `GetTables() []string` should be enough?

Reply:
Yet